src/app/data/pontusx-account-names.ts

```diff
@@ -1,65 +1,58 @@
-import axios from 'axios'
-import { useQuery, UseQueryOptions } from '@tanstack/react-query'
+import { usePontusXRegistry } from '@deltadao/pontusx-registry-hooks'
+import { UseQueryOptions } from '@tanstack/react-query'
+import { useGetRuntimeAccountsAddresses } from '../../oasis-nexus/api'
+import { Network } from '../../types/network'
+import { hasTextMatchesForAll } from '../components/HighlightedText/text-matching'
+import { getOasisAddress } from '../utils/helpers'
 import {
-  AccountMetadata,
   AccountMap,
+  AccountMetadata,
   AccountMetadataInfo,
   AccountNameSearchRuntimeMatch,
   AccountNameSearchRuntimeResults,
 } from './named-accounts'
-import { useGetRuntimeAccountsAddresses } from '../../oasis-nexus/api'
-import { Network } from '../../types/network'
-import { hasTextMatchesForAll } from '../components/HighlightedText/text-matching'
-import { getOasisAddress } from '../utils/helpers'
-import * as externalLinks from '../utils/externalLinks'
 
 type PontusXAccountsMetadata = {
   map: AccountMap
   list: AccountMetadata[]
 }
 
-const getPontusXAccountsMetadata = async (): Promise<PontusXAccountsMetadata> => {
-  const response = await axios.get(externalLinks.api.deltadao_named_addresses)
-  if (response.status !== 200) throw new Error("Couldn't load names")
-  if (!response.data) throw new Error("Couldn't load names")
+export const usePontusXAccountsMetadata = () => {
+  const { isLoading, error, data } = usePontusXRegistry({
+    includeDeprecated: true,
+  })
+
+  if (error) {
+    console.log('Failed to load Pontus-X account names', error)
```

Comment:
issue: use `console.error()` for error log
```suggestion
    console.error('Failed to load Pontus-X account names', error)
```